client/settings/fraud-protection/protection-levels/index.tsx

```diff
@@ -183,7 +183,6 @@ const ProtectionLevels: React.FC = () => {
 							level={ ProtectionLevel.BASIC }
 						/>
 					</li>
-					<hr className="fraud-protection__list-divider" />
```

Comment:
Not semantically correct to use an `hr` between `li`, it's creating invalid markup